src/Controls/tests/Shared.UITests/Tests/Issues/Issue17366.cs

```diff
@@ -16,11 +17,9 @@ public Issue17366(TestDevice device) : base(device)
 		[Category(UITestCategories.Brush)]
 		public void Issue17366Test()
 		{
-			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows },
-					"The bug only happens on iOS; see https://github.com/dotnet/maui/pull/17789");
-
```

Comment:
Should we keep these messages as comments? I see a few have been deleted, and they usually link to issues that we need for more context.